src/translationmanager.js

```diff
@@ -0,0 +1,31 @@
+import { format } from './utils';
+import getTranslationsJSON from './translations';
+
+export default class TranslationManager {
+    constructor(instance) {
+        this.instance = instance;
+        this.language = this.instance.language;
+
+        this.translations = getTranslationsJSON();
+    }
+
+    addCustomTranslations(translations) {
+        this.translations = Object.assign(this.translations, translations);
+    }
+
+    translate(str, args) {
+        let translation = (this.translations[this.language] && this.translations[this.language][str]) || str;
+
+        if (typeof translation === 'object') {
+            translation = args && args.count ?
+                this.getPluralizedTranslation(translation, args.count) :
+                str;
+        }
+
+        return format(translation, args || {});
+    }
+
+    getPluralizedTranslation(translations, count) {
+        return translations[count] || translations['default'];
```

Comment:
Does this still work, with `"0"` (string) in the translations JSON and `0` (number) in the parameter?

Reply:
- Yes that works are expected, the  string gets converted to a number
- ![image](https://user-images.githubusercontent.com/7310479/146763218-44e9d09d-1c70-48b8-b528-d4a7347239a6.png)
- ![image](https://user-images.githubusercontent.com/7310479/146763174-c61ce789-d6f2-44d2-846b-b98c7feeeb50.png)